misc/docker-integration/integration_test.go

```diff
@@ -57,27 +57,38 @@ func runSuite(t *testing.T, tempdir string) {
 	require.True(t, acc.Coins.IsAllGTE(minCoins),
 		"test1 account coins expected at least %s, got %s", minCoins, acc.Coins)
 
-	// add gno.land/r/demo/tests package as tests_copy
+	// add gno.land/r/demo/helloworld/world
+	// TODO: it should be added automatically by its parent package `gno.land/r/demo/helloworld`.
 	dockerExec(t,
 		`echo 'pass' | gnokey maketx addpkg -insecure-password-stdin \
 			-gas-fee 1000000ugnot -gas-wanted 2000000 \
 			-broadcast -chainid dev \
-			-pkgdir /opt/gno/src/examples/gno.land/r/demo/tests/ \
-			-pkgpath gno.land/r/demo/tests_copy \
+			-pkgdir /opt/gno/src/gnovm/tests/integ/helloworld/world/ \
 			-deposit 100000000ugnot \
 			test1`,
 	)
-	// assert gno.land/r/demo/tests_copy has been added
+
+	// add gno.land/r/demo/helloworld
+	dockerExec(t,
+		`echo 'pass' | gnokey maketx addpkg -insecure-password-stdin \
+			-gas-fee 1000000ugnot -gas-wanted 2000000 \
+			-broadcast -chainid dev \
+			-pkgdir /opt/gno/src/gnovm/tests/integ/helloworld/ \
+			-deposit 100000000ugnot \
+			test1`,
+	)
```

Comment:
we should start making a simple command builder taking a struct with typed fields for this soon.

Reply:
Will do this soon. Maybe in a separate PR.